tests/conftest.py

```diff
@@ -8,6 +8,11 @@ def get_dataset(filename):
     return CFDataset(resource_filename(__name__, 'data/tiny_{}.nc').format(filename))
 
 
+@fixture
+def tiny_filepath(request):
+    return resource_filename(__name__, 'data/tiny_{}.nc').format(request.param)
```

Comment:
Nice! I wonder if you can DRY up `tiny_dataset` now by using this fixture in it. Not a big deal (a few characters of code), but doing a thing only in one place means you only have to fix it once if you make a mistake.